pages/linux/toilet.md

```diff
@@ -0,0 +1,19 @@
+# toilet
+
+> A tool to display ASCII-art fonts.
+
+- Generate by directly inputting text:
```

Comment:
I feel like we should let explicit what exactly is this generating. Maybe: 
```suggestion
- Generate a art by directly inputting text:
```

Reply:
Perhaps:
>Generate an ASCII art image by directly inputting text:

Reply:
even better @pxgamer